README.md

```diff
@@ -242,7 +242,36 @@ class Sample:
 
 See [marshmallow's documentation about extending `Schema`](https://marshmallow.readthedocs.io/en/stable/extending.html).
 
-### Custom NewType declarations
+### Custom type aliases
+
+This library allows you to specify [customized marshmallow fields](https://marshmallow.readthedocs.io/en/stable/custom_fields.html#creating-a-field-class) using python's Annoted type [PEP-593](https://peps.python.org/pep-0593/).
+
+```python
+from typing import Annotated
```

Comment:
Is it worth documenting that for python 3.8 one must import `Annotated` from `typing_extensions`?

E.g.

https://github.com/lovasoa/marshmallow_dataclass/blob/dbd909f32eaf85ed2d9e7442e291c193e9a45d30/marshmallow_dataclass/__init__.py#L74-L77

Reply:
Documented